lib/builders/restoreArgBuilder.js

```diff
@@ -25,7 +25,7 @@ module.exports = (value) => {
     if (value.noIncremental) {
         args = args.concat(['--no-incremental']);
     }
-    if (value.noDepenencies) {
```

Comment:
is this a bug fix?

Reply:
if it is can you adjust the initial comment in this PR and add this bug to the list of things this PR comes with?

Reply:
Yeah, you are right, it was a typo but also a bug

Reply:
done!